lib/screens/settings/settings_screen.dart

```diff
@@ -210,7 +212,7 @@ class _SettingsState extends State<Settings> {
                           context,
                           MaterialPageRoute(
                               fullscreenDialog: true,
-                              builder: (context) => SentryFlutterScreen()),
+                              builder: (context) => SentrySDKDebugScreen()),
```

Comment:
Not really relevant to this PR, but `MaterialPageRoute` has a `settings` argument. If that's set, it'll be recorded as additional information in the navigation breadcrumbs.